src/index.js

```diff
@@ -92,11 +92,11 @@ document.addEventListener('DOMContentLoaded', () => {
   renderTaskList();
 });
 
-// const clearButton = document.getElementById('clear-btn');
-// clearButton.addEventListener('click', () => {
-//   tasks = tasks.filter((task) => !task.completed);
-//   localStorage.setItem('tasks', JSON.stringify(tasks));
-//   renderTaskList();
-// });
+const clearButton = document.getElementById('clear-btn');
+clearButton.addEventListener('click', () => {
+  tasks = tasks.filter((task) => !task.completed);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTaskList();
+});
```

Comment:
Great job adding the clear completed functionality 👏🏾 

[REQUIRED] However, the project requirements state that a new file should be created for all the functionalities of this milestone. You are expected to make use of modules (which is using `export` to export functions to other files, and `import` to retrieve functions from other files).

![image](https://user-images.githubusercontent.com/38088818/230709405-057601a2-93da-4255-9df3-e8332f6c4f51.png)

[OPTIONAL] Ideally, the `add`, `edit`, and `remove` functionalities should also be in another file and imported to the `index.js` file as modules. This should have been required from the previous milestone, I highly recommend that you take it into consideration.